src/predict/jobs/forecast/Dockerfile

```diff
@@ -10,30 +10,18 @@ WORKDIR /app/
 RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
 
-# Production - daily forecasts
-FROM base as production_daily
+# Production
+FROM base as production
 ENV ENV=production
-CMD ["python", "daily_forecasts.py"]
+CMD ["sh", "run_scripts.sh"]
```

Comment:
@123MwanjeMike . I tried to have 2 CMDs for each script but got the warning. `There can only be one CMD instruction in a Dockerfile or build stage. Only the last one will have an effect` so I've just used a bash script. Don't know why I didn't consider that first. 😅🤡🤦🏾‍♂️

Reply:
Thanks for this improvement @Mnoble-19 
For the 3-5minutes timespan, the VMs run for 8minutes by default so we are good.